dts/bindings/sensor/invensense,mpu9250.yaml

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+description: |
+    InvenSense MPU-9250 Nine-Axis (Gyro + Accelerometer + Compass). See more
+    info at https://www.invensense.com/products/motion-tracking/9-axis/mpu-9250/
+
+compatible: "invensense,mpu9250"
+
+include: i2c-device.yaml
+
+properties:
+    irq-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INT signal default configuration is active-high.  The
+        property value should ensure the flags properly describe the
+        signal that is presented to the driver.
+
+    gyro-sr-div:
+      type: int
+      required: false
+      default: 99
+      description: |
+        Default gyrscope sample rate divider. This divider is only effective
+        when gyro-dlpf is in range 5-184.
+        rate = sample_rate / (1 + gyro-sr-div)
+        Valid range: 0 - 255
+
+    gyro-dlpf:
+      type: int
+      required: false
+      default: 5
```

Comment:
this and others: defaults need to be justified (e.g. set to sensor defaults), otherwise, make them required.

Reply:
Removed defaults